packages/eas-cli/src/commands/build/list.ts

```diff
@@ -51,6 +52,7 @@ export default class BuildList extends EasCommand {
     ...EasPaginatedQueryFlags,
     limit: getLimitFlagWithCustomValues({ defaultTo: 10, limit: BUILDS_LIMIT }),
     ...EasNonInteractiveAndJsonFlags,
+    simulator: Flags.boolean(),
```

Comment:
Maybe we should also enforce that this is only applicable when using the `ios` value for the `--platform` flag? What do you think?